setup.py

```diff
@@ -34,6 +34,18 @@ def read_file(fname):
 is_python_27 = python_version == (2, 7)
 is_python_2 = python_version[0] == 2
 
+pytest_version = "pytest==4.6.6" if is_python_27_or_greater else "pytest<3.3"
```

Comment:
inline versions made requirements harder to read below, so assigning them here intead.

Reply:
`is_python_2` doesn't look like it's needed any more, I know it's not part of your change though.

Reply:
👍